src/components/landing/ourmission.tsx

```diff
@@ -0,0 +1,23 @@
+const OurMission = () => {
+  return (
+    <div className="mr-42 flex flex-col items-end justify-center text-right">
+      <p className="text-pse-purple-300 font-pse-crimson-text text-5xl">
```

Comment:
```suggestion
      <p className="text-pse-purple-400 font-pse-crimson-text text-5xl">
```

I added an extra color that was missing so the shades got adjusted